cmd/minikube/cmd/config/validations.go

```diff
@@ -50,6 +51,14 @@ func IsValidDiskSize(name string, disksize string) error {
 	return nil
 }
 
+func IsValidURL(name string, location string) error {
+	_, err := url.Parse(location)
```

Comment:
Fair point.  This will probably break downstream if you put something without a scheme.  Maybe the behavior for strings without a scheme should be to assume its `file:///$PWD?`